data/css/escola.scss

```diff
@@ -203,3 +254,52 @@ where it needs to be */
   min-height: 102px;
   background-color: #323233;
 }
+
+scrollbar {
+    trough {
+      margin: 0;
+        padding: 0px 3px;
+        background-color: transparentize(black, 1 - 0.90);
+        background-image: linear-gradient(89deg,
+                                          transparentize(white, 1 - 0.1) 0%,
+                                          transparentize(black, 1 - 0.1) 100%);
+        box-shadow: inset 1px 0px 0px 0px transparentize(black, 1 - 0.15);
+    }
+
+    slider {
+        background-color: transparentize(white, 1 - 0.5);
+    }
+}
+
+.composite {
+  .home-page {
+    .landing {
+      .background {
+        min-height: 250px;
+      }
+
+      .BannerDynamic {
+        &__logo {
+          min-height: 120px;
+          min-width: 400px;
+          font-size: 60px;
```

Comment:
`font-size` doesn't really affect the size of the DynamicLogo,the size you see is only the result of the min-height and min-width you use there.